core/src/main/scala/com/criteo/cuttle/App.scala

```diff
@@ -211,64 +226,77 @@ private[cuttle] case class App[S <: Scheduling](project: CuttleProject[S],
       val jobIds = parseJobIds(jobs)
       val limit = Try(l.toInt).toOption.getOrElse(25)
       val offset = Try(o.toInt).toOption.getOrElse(0)
-      val asc = (a.toLowerCase == "asc")
+      val asc = a.toLowerCase == "asc"
       val ids = if (jobIds.isEmpty) allIds else jobIds
 
-      def getExecutions() = kind match {
+      def getExecutions: IO[Option[(Int, Seq[ExecutionLog])]] = kind match {
         case "started" =>
-          Some(
-            executor.runningExecutionsSizeTotal(ids) -> executor
-              .runningExecutions(ids, sort, asc, offset, limit))
+          IO.pure(
+            Some(
+              executor.runningExecutionsSizeTotal(ids) -> executor
```

Comment:
I'm not sure this will be executed more than once

Reply:
Yes IO.pure is evaluated directly. Use IO.apply instead